modules/default/newsfeed/fetcher.js

```diff
@@ -46,6 +46,7 @@ var Fetcher = function(url, reloadInterval, encoding) {
 			var title = item.title;
 			var description = item.description || item.summary || item.content || '';
 			var pubdate = item.pubdate || item.published || item.updated;
+			var url = item.url || item.link;
```

Comment:
Maybe this should be changed to:

````
var url = item.url || item.link || '';
````

This prevents issues with feeds that omit both fields.